Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM alpine:3.4
+FROM alpine:3.6
 
 ENV GOPATH /go
 
-RUN apk update && apk add ca-certificates go git && \
+RUN apk update && apk add ca-certificates go git musl-dev && \
```

Comment:
I would change to `apk add --no-cache`  and remove the `rm -rf /var/cache/apk/*` lines (there are two of them)